core/src/main/kotlin/org/sert2521/sertain/motors/MotorController.kt

```diff
@@ -0,0 +1,257 @@
+package org.sert2521.sertain.motors
+
+import org.sert2521.sertain.units.*
+import com.ctre.phoenix.motorcontrol.can.TalonSRX as CtreTalon
+import com.ctre.phoenix.motorcontrol.ControlMode as CtreControlMode
+import com.ctre.phoenix.motorcontrol.NeutralMode as CtreNeutralMode
+
+class MotorController<T : MotorId>(
+        val id: T,
+        vararg followerIds: MotorId,
+        val name: String = "ANONYMOUS_MOTOR",
```

Comment:
What's name used for? Debug eventually?

Reply:
Yeah, debugging. If it turns out to be useless we can remove it.